### PR TITLE
0.11.0

### PR DESCRIPTION
**Pull request by NathanPJF**
### What are you trying to accomplish with this PR?

Merging the 0.11.0 milestone into the master branch.

Documentation updates on their way.

### Checklist
For contributors:
- [x] I have [updated the docs](https://github.com/Shopify/slate/blob/master/CONTRIBUTING.md#documentation) to reflect these changes, if applicable.

For maintainers:
- [ ] I have :tophat:'d these changes.
- [ ] I have bumped the `package.json` version in a separate PR, if applicable.

### CODE REVIEW

File: config-sample.yml [hunk 1]
```diff
@@ -15,12 +15,14 @@
 
 development:
   password:
-  theme_id:
```
Comment (chrisberthe) [inline config-sample.yml]: 🤦‍♂️

File: config-sample.yml [hunk 2]
```diff
@@ -15,12 +15,14 @@
...
+    #- settings_data.json # Uncomment this line to avoid resetting theme settings
 
 production:
   password:
   theme_id: "live"
   store: your-shop.myshopify.com
-  ignore_files:
\ No newline at end of file
+  ignore_files:
+    #- settings_data.json # Uncomment this line to avoid resetting theme settings
```
Comment (chrisberthe) [inline config-sample.yml]: @NathanPJF what's the verdict about uncommenting these?

**Comment by chrisberthe** [comment]
Note: there needs to be an update to the documentation as well.

File: config-sample.yml (hunk 2 above)
Comment (NathanPJF) [inline config-sample.yml]: Verdict is: merging as comment for now and going to do a couple more tests as a new issue.

**Pull request merged**